README.md

```diff
@@ -38,11 +38,42 @@ NOTE: by default the waggle-dance-rpm module will build an RPM artifact for inst
 
 ## Installing
 
-It is intended that Waggle Dance is run as a service serving as a proxy to different metastores. The primary configured metastore is the only one to which you may also write data via Hive. It can either be installed on an existing EMR instance (for example the EMR cluster of your primary metastore) or on a separate EC2 instance.
+Waggle Dance comes in two binaries distributions: TGZ and RPM. It can either be installed on an existing EMR instance (for example the EMR cluster of your primary metastore) or on a separate EC2 instance.
```

Comment:
Should we mention anything AWS specific? It could be run anywhere and I wouldn't want to put people off who wanted to install it on something else (e.g. on premise nodes, GCP etc.) Perhaps we could say 

"It is intended to be installed as a service available on a machine that is accessible from wherever you want to query it from (and with access to the Hive metastore service(s) that it is federating). If you are using AWS this could be on an existing EMR instance (for example the EMR cluster of your primary Hive  metastore) or on a separate EC2 instance. It would be deployed in a similar fashion for other cloud or internal platforms."